### ISSUE TITLE
feature request: eve app, history, graphs

### ISSUE THREAD

**Issue by klaus-schuster** [jensweigele/ioBroker.yahka#63]
see 

https://forum.iobroker.net/viewtopic.php?t=14953

**Comment by PhilippLeitner** [comment]
+1

**Comment by PhilippLeitner** [comment]
Gibt es hierfür schon einen Ansatz?

**Comment by jensweigele** [comment]
Bezüglich History/Graph: Ich hab es mir schonmal angeschaut. Sieht nach größerem Aufwand aus. 

Mit dem aktuellen Master sollte es möglich sein, die Energiewerte anzuzeigen. Ich teste das morgen mal

**Comment by PhilippLeitner** [comment]
Kann man die History vielleicht für Die Temperatur einbauen? Ich denke das würde vielen Usern gefallen.

**Comment by schroedingerscat8** [comment]
> Mit dem aktuellen Master sollte es möglich sein, die Energiewerte anzuzeigen.
Das Anzeigen von Energiewerten ist eine Funktion nach der ich schon lange suche. Wie können Energiewerte angezeigt werden?

**Comment by PhilippLeitner** [comment]
Gibt es hierzu schon Neuigkeiten?

**Comment by jensweigele** [comment]
@PhilippLeitner : von meiner Seite gibt es keine Neuigkeiten - wie oben beschrieben ist es ein größerer Aufwand den ich aktuell nicht leisten kann

@schroedingerscat8 : du musst einen Service über die "CommunityTypes" hinzufügen. Ich habe gerade keine Yahka-Instanz da, aber es gibt einen Service der heißt "CommunityTypes: Energy" oder so.
Sichtbar ist dieser Wert aber nur in bspw. der Eve-App. Die HomeKit App unterstützt diese Services/Charakteristiken nicht.

**Comment by schroedingerscat8** [comment]
@jensweigele leider finde ich CommunityType: energy nicht, wenn ich einem Gerät einen Service hinzufügen möchte.
Ich könnte zwar zB. AudioDeviceService, PlaybackDeviceService, ... bis zu NotificationService (in Summe stehen ca. 13 CommunityTypes zur Auswahl) - aber eben keinen "CommunityTypes: Energy".

**Comment by PhilippLeitner** [comment]
@jensweigele leider werden die CommunityTypes noch immer nicht angezeigt.

**Comment by SpeedmaxX** [comment]
Hallo @PhilippLeitner  @jensweigele , 

gibt es hier schon eine Lösung? Den CT "energy" oder ähnlich, kann ich leider auch noch nicht finden.

VG - SM

**Comment by appinator** [comment]
> Hallo @PhilippLeitner @jensweigele ,
> 
> gibt es hier schon eine Lösung? Den CT "energy" oder ähnlich, kann ich leider auch noch nicht finden.
> 
> VG - SM

Der fehlt bei mir auch. :/

**Comment by nicoh88** [comment]
+1

**Comment by nicoh88** [comment]
@jensweigele 

Hi Jens, ich habe damals für [homebridge-devolo die FakeGate-Integration](https://github.com/kdietrich/homebridge-devolo/tree/master/pictures/fakegato) programmiert. Würde dir gerne dabei helfen, dass hier umzusetzen - weil dann könnte ich auch auf ioBroker umsteigen. 👍 

Ich bin kein guter Programmierer, ich kann bestehenden Code _leicht_ erweitern und könnte mich um die **Logik** kümmern. Du kannst dir ja mal die FakeGato-Integration bei [homebridge-devolo für den Kontaktsensor](https://github.com/kdietrich/homebridge-devolo/blob/master/dist/devices/HBDevoloDoorWindowDevice.js) anschauen -  suche nach  `// START FakeGato (eve app)`, habe eigentlich alles ordentlich dokumentiert. 
Jedoch weiß ich so garnicht, wo ich bei `ioBroker.yahka` anfangen soll ... der erste Schritt wäre, jeder Kontaktsensor, soll bestimmte Werte über [fakegato-history](https://github.com/simont77/fakegato-history) speichern. 

Gruß Nico

**Comment by jensweigele** [comment]
@nicoh88 über https://github.com/jensweigele/ioBroker.yahka/issues/120 kann man ja nun die aktuellen Stromwerte anzeigen.
Ist dieses Issue noch relevant?
Gibt es eine Möglichkeit nun über Eve, die History erstellen zu lassen oder sollte Yahka die History aus ioBroker ausgeben (über den ioBroker History Adapter)?

**Comment by nicoh88** [comment]
@jensweigele ja, dass Issue ist noch relevant. Die ganzen Graphen fehlen - „nur“ der momentane Stromverbrauch wird angezeigt

Genau, Yahka sollte die History aus iobroker ausgeben und so wie fakegato oder für fakegato aufbereiten.

Gruß

**Comment by achimmm** [comment]
@jensweigele , @nicoh88 hallo zusammen, würde dieses Feature ebenfalls sehr begrüßen - ist aktuell der letzte, verbleibende Grund, auf FHEM zu bleiben & nicht auf ioBroker umzustellen.

**Comment by jensweigele** [comment]
Ich bin da aktuell dran - wird aber noch bisschen dauern.

**Comment by MrDrache333** [comment]
Any updates on this one? I would love to see this feature. Also interesting would be to know how the data could be added...via json-Array with Data and Value and the collection of the Data has to be done manually or some automatic collection?

**Comment by jhubig** [comment]
+1

Would be great to be also ably to link it to one of the history adapters. Thanks.

**Comment by DanielPiralli** [comment]
Would be great, if this feature will find its way into yahka. Any news?

**Comment by virus18** [comment]
@jensweigele 
Moin Jens, kann man wohl noch mit der Implementierung rechnen oder liegt die auf Eis ?
